app/controllers/contact/govuk_app_controller.rb

```diff
@@ -7,9 +7,9 @@ def create
     end
 
     if type == "problem"
-      redirect_to contact_govuk_app_problem_reports_path
```

Comment:
It might be a bit better if this commit were fixed up so that incorrect paths/content/etc were not in the commit history?

Reply:
Sure, I've split the commit up into the various commits 👍